src/Component/JsonSchema/Generator/Runtime/data/Normalizer/ValidationException.php

```diff
@@ -2,7 +2,7 @@
 
 use Symfony\Component\Validator\ConstraintViolationListInterface;
 
-class ValidationException extends \RuntimeException
+class ValidationException extends RuntimeException
```

Comment:
It won't work since we don't have the `use` statements above and no `RuntimeException` exists within this namespace.